docs/en/reference/generating-migrations.rst

```diff
@@ -260,7 +260,13 @@ with a schema filter.
 
 .. code-block:: php
 
-    $connection->getConfiguration()->setFilterSchemaAssetsExpression("~^(?!t_)~");
+    $connection->getConfiguration()->setSchemaAssetsFilter(static function (string|AbstractAsset $assetName): bool {
+        if ($assetName instanceof AbstractAsset) {
+            $assetName = $assetName->getName();
+        }
+
+        return preg_match("~^(?!t_)~", $assetName);
```

Comment:
`preg_match()` returns an integer. I believe you'll have to cast it in order to make this example work with strict types enabled.

```suggestion
        return (bool) preg_match("~^(?!t_)~", $assetName);
```